Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -448,8 +409,7 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
     /// - parameter callDelegate: Should the delegate be called for this change
     @objc open func highlightValue(x: Double, y: Double, dataSetIndex: Int, callDelegate: Bool)
     {
-        guard let data = _data else
-        {
+        guard let data = data else {
```

Comment:
regression code style. Please check #3034 for code style issues and update here? If you can manage to reset hard with #3034, we can merge #3034 so you won't have to do dup work. up to you